user/troubleshooting/vpn-troubleshooting.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: doc
+title: VPN Troubleshooting
+permalink: /doc/vpn-troubleshooting/
+---
+
+# VPN Troubleshooting Guide #
+
+## Tips 
+
+* Check the VPN service's log in the VPN VM by running:
+    ~~~
+    sudo journalctl -u qubes-vpn-handler
+    ~~~
+* Always test your basic VPN connection before adding scripts. 
+
+* Test DNS: Ping a familiar domain name from an appVM. It should print the IP address for the domain.
+
+* Use `iptables -L -v` and `iptables -L -v -t nat` to check firewall rules. The latter shows the critical PR-QBS chain that enables DNS forwarding.
+
+## VPN does not reconnect after suspend 
+After suspend/resume, your VPN may not automatically reconnect. In order to get it to work, you must kill your VPN system and restart it. 
```

Comment:
What do you mean by "kill your VPN system"? The VPN VM? Restart the whole system? I think the original was about openvpn process.